src/puck.ts

```diff
@@ -599,30 +617,51 @@ export class LookupPuck {
 
   unmount(): void {
     removePuck();
-    this.disable();
+    this.setEnabledState(LookupPuckEnabledState.disabled);
     this.puck = undefined;
   }
 
-  enable(): void {
-    this.enabled = true;
-    this.safeAreaProvider.delegate = this;
-    if (this.puck) {
-      this.puck.addEventListener('pointerdown', this.onPuckPointerDown);
+  setEnabledState(enabledState: LookupPuckEnabledState): void {
+    const previousState = this.enabledState;
+    this.enabledState = enabledState;
```

Comment:
Alternatively, I could set the `this.enabledState` just before returning each time?